r/R/feather.R

```diff
@@ -147,14 +147,17 @@ read_feather <- function(file, col_select = NULL, as_data_frame = TRUE, ...) {
     file <- make_readable_file(file)
     on.exit(file$close())
   }
-  reader <- FeatherReader$create(file, ...)
+  reader <- FeatherReader$create(file)
```

Comment:
Is the dropping of `...` here intentional?

Reply:
@jonkeane yes! it's intentional

Reply:
should I re-add it?

Reply:
No, you don't need to re-add it since we took all of those out, but if this is the only place the `...` is used in this function, we should adjust its signature + docs too.

Reply:
The dots should be passed to `make_readable_file()`

Reply:
for example, from https://github.com/apache/arrow/pull/9756/files
I still could still pass a mmap = TRUE to FileCreate from read_feather, so the ... is probably valuable in this context